apps/token/src/routes/governance/components/proposals-list/proposals-list.tsx

```diff
@@ -44,7 +47,16 @@ export const ProposalsList = ({ proposals }: ProposalsListProps) => {
 
   return (
     <>
-      <Heading title={t('pageTitleGovernance')} />
+      <div className="grid xs:grid-cols-2 items-center">
+        <Heading centerContent={false} title={t('pageTitleGovernance')} />
+        <Link
```

Comment:
chore (non-blocking): can we add some basic tests to validate this heading and link are rendered?

Reply:
Sure